components/application-broker/internal/mapping/controller.go

```diff
@@ -67,10 +71,16 @@ func New(emInformer cache.SharedIndexInformer,
 	log logrus.FieldLogger,
 	livenessCheckStatus *broker.LivenessCheckStatus) *Controller {
 
+	queue := workqueue.NewRateLimitingQueue(
+		workqueue.NewMaxOfRateLimiter(
+			workqueue.NewItemExponentialFailureRateLimiter(200*time.Millisecond, 120*time.Second),
```

Comment:
I changed the queue configuration accordingly to the [changes](https://github.com/kyma-project/kyma/commit/d5f6e5fbe95d69d5ab5445fc472c822d952f9a00) in SBUC. The application mapping has no child-resource dependencies (like SBU) so we can easily bump the reconcile time. It should improve performance on highly loaded clusters.